src/views/NutritionView.vue

```diff
@@ -283,6 +286,8 @@ export default {
         };
       }
       this.currentProductData = data;
+      this.nutritionData = await this.getNutritionData();
+      await this.setNutritionData();
```

Comment:
This function is not asynchronous, so you don't need the `await`
```suggestion
      this.setNutritionData();
```